### PR TITLE
Fixed#10485 search results overlaps with empty inbox placeholder text, sign…

### PR DESCRIPTION
**Pull request by Dayakargoud**
### First time contributor checklist

- [x] I have read [how to contribute](https://github.com/signalapp/Signal-Android/blob/master/CONTRIBUTING.md) to this project
- [x] I have signed the [Contributor License Agreement](https://whispersystems.org/cla/)

### Contributor checklist

- [x] I am following the [Code Style Guidelines](https://github.com/signalapp/Signal-Android/wiki/Code-Style-Guidelines)
- [x] I have tested my contribution on these devices:
 * OPPO A52 , Android 11
 * REDMI4, Android 7.1.2
- [x] My contribution is fully baked and ready to be merged as is
- [x] I ensure that all the open issues my contribution fixes are mentioned in the commit message of my first commit using the `Fixes #1234` [syntax](https://help.github.com/articles/closing-issues-via-commit-messages/)

----------

### CODE REVIEW

File: app/src/main/java/org/thoughtcrime/securesms/conversationlist/ConversationListFragment.java [hunk 1]
```diff
@@ -557,6 +560,7 @@ private void onSearchResultChanged(@Nullable SearchResult result) {
     searchAdapter.updateResults(result);
 
     if (result.isEmpty() && activeAdapter == searchAdapter) {
+      if (emptyState.resolved()) emptyState.get().setVisibility(View.GONE);
```
Comment (ankit167) [inline app/src/main/java/org/thoughtcrime/securesms/conversationlist/ConversationListFragment.java:563]: Suggest to add blocks for better readability

Comment (ankit167) [inline app/src/main/java/org/thoughtcrime/securesms/conversationlist/ConversationListFragment.java:563]: Suggest to add blocks for better readability

Comment (Dayakargoud) [reply app/src/main/java/org/thoughtcrime/securesms/conversationlist/ConversationListFragment.java]: Ok, I added blocks, please take a look.

**Comment by ankit167** [comment]
The issue is resolved already. This PR can be closed

**Pull request closed without merging**